### PR TITLE
Task/01 setup

### PR DESCRIPTION
**Pull request by tetiana-zhuchenko**
Acceptance criteria for 1

- [x]  Mentor is added to pr
- [x]  AC copied to pr description
- [x] Link to deployed version added to pr
- [ ]  About block (text and image) added to the main page
- [x] On the widths 390px and 1280px, the site layout corresponds to the design

Acceptance criteria for 1*
- [x] Created separate component About
- [x]  Its styles added to [.module.css](https://github.com/css-modules/css-modules)
- [ ]  For App.tsx styles moved to App.module.css

### CODE REVIEW

**Comment by polishTania** [comment]
Everithing good

File: src/index.css [hunk 1]
```diff
@@ -1,71 +1,20 @@
...
+  --main-bg: var(--gray-4);
+  --main-font-color: var(--darkest-gray);
+  --main-red: rgba(239, 73, 52, 1);
+  --hover-red: rgba(212, 56, 36, 1);
+  --darkest-gray: rgba(17, 17, 17, 1);
+  --gray-1: rgba(51, 51, 51, 1);
+  --gray-2: rgba(101, 101, 101, 1);
+  --gray-3: rgba(204, 204, 204, 1);
+  --gray-4: rgba(225, 225, 225, 1);
+  --gray-hover: rgba(241, 241, 241, 1);
```
Comment (polishTania) [inline src/index.css:11]: it's a good idea to make variables

**Review by polishTania — COMMENTED** [review]
test

File: src/index.css (hunk 1 above)
Comment (tetiana-zhuchenko) [reply src/index.css:11]: Thanks

**Review by polishTania — APPROVED** [review]

**Pull request merged**